frontend/gulp/server.js

```diff
@@ -39,6 +39,7 @@ function browserSyncInit(baseDir, browser) {
     startPath: '/',
     server: server,
     ghostMode: false,
+    https: true,
```

Comment:
we had this line before, i've just put it back
by default browsersync provides a self-signed certificate which needs to be
accepted in chrome

On Tue, May 17, 2016 at 4:30 PM, Edo Shor <notifications@github.com> wrote:

> In frontend/gulp/server.js
> <https://github.com/Bnei-Baruch/galaxy2/pull/50#discussion_r63532385>:
>
> > @@ -39,6 +39,7 @@ function browserSyncInit(baseDir, browser) {
> >      startPath: '/',
> >      server: server,
> >      ghostMode: false,
> > +    https: true,
>
> This means we have to have our own cert files on our local dev machine ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Bnei-Baruch/galaxy2/pull/50/files/6e93b810e2b47a33e15105bca5b04e84243d11c6#r63532385>
>



-- 
Kind regards,
Konstantin Alexandrov

http://illuzia.net